app/controllers/worldwide_organisations_controller.rb

```diff
@@ -60,4 +88,28 @@ def secondary_role
   def office_roles
     @worldwide_organisation.office_staff_roles.map { |office_staff| RolePresenter.new(office_staff, view_context) }
   end
+
+  def test_helper_content
+    ab_test_helper = WorldwideAbTestHelper.new
+    ab_test_helper.content_for(@world_location.slug) if ab_test_helper.has_content_for?(@world_location.slug)
+  end
+
+  def embassies_test_data
+    test_helper_content["embassies"] if test_helper_content
+  end
+
+  def embassy_test_data(embassy)
+    embassies_test_data[embassy][0] if embassies_test_data
+  end
+
+  def configure_ab_variant
+    ab_test = GovukAbTesting::AbTest.new("WorldwidePublishingTaxonomy", dimension: 45)
+    @requested_variant = ab_test.requested_variant(request.headers)
+    @requested_variant.configure_response(response)
+  end
+
+  def show_b_variant?
+    configure_ab_variant if @requested_variant.nil?
```

Comment:
Wouldn't this always be `nil`? Wondering if we need the `if` clause...

Also there's a bit of duplication of setting up AB test stuff (see method `redirect_if_required_for_ab_test` use in the `show` controller method). I added a before_action where I do all ab test setup in https://github.com/alphagov/whitehall/blob/master/app/controllers/world_location_news_controller.rb#L35-L39, not sure if that's neater?